tests/cogs/base/test_cog.py

```diff
@@ -192,7 +192,7 @@ async def test_invalid_clear(base_cog: BaseCog):
 async def test_load_cog(base_cog: BaseCog):
     with mock.patch.object(discord.ext.commands.bot.Bot, 'load_extension') as mock1:
         await dpytest.message(koalabot.COMMAND_PREFIX + "load_cog BaseCog")
-    mock1.assert_called_with('koala.cogs.BaseCog')
+    mock1.assert_called_with(".BaseCog", package="koala.cogs")
```

Comment:
It fails if I change this?

Reply:
You'll need to change the command above to base too. Also you need to import setup into the __init__.py of each extension from the cog.py, otherwise the load extension won't work